### DISCUSSION TITLE
Missing "Triage" role in base permissions pettings

### DISCUSSION THREAD

**Discussion by Deep0Thinking** [community/community#121924] [Repositories]
### Select Topic Area

Bug

### Body

I noticed that the **Triage** role, which is listed under pre-defined repository roles, is **not available** in the base permissions settings for organization members. Could this be an oversight or is there another way to assign the Triage role that I might be missing? Thank you for clarifying.

**Pre-defined roles**:
<img width="944" alt="Screenshot 2024-05-02 at 1 46 56 AM" src="https://github.com/community/community/assets/103571424/1ac05bb2-a53c-45dc-bc22-c32b4b8502ed">

**Base permissions**:
<img width="941" alt="Screenshot 2024-05-02 at 1 47 36 AM" src="https://github.com/community/community/assets/103571424/7f66c654-6fb0-43bb-9f4c-eb7100e97d47">

**Comment by steferd6** [comment]
If you're seeing "Missing 'Triage' role in base permissions settings", it usually means that a specific user or team doesn’t have the required permission to manage or prioritize issues in your system — often seen in platforms like GitHub or project management tools. This can be resolved by updating the role configuration under repository or workspace settings.

Interestingly, similar to how permissions must be assigned properly to function effectively, the [Fluorocholine PET Scan](https://centrechorfi.com/pet-scan/) also relies on precise targeting — it’s designed to identify metabolic activity in prostate cancer by tracking choline uptake. Just like roles need to be assigned correctly in software to avoid workflow errors, accurate targeting in diagnostic imaging ensures meaningful results.
